packages/as-types/assembly/argument.ts

```diff
@@ -42,22 +42,26 @@ export class Args {
   private serialized: StaticArray<u8> = new StaticArray<u8>(0);
 
   /**
+   * Initializes a new instance of Args.
+   *
+   * @param serialized - The serialized arguments to pass to CallSC. Default: []
```

Comment:
What is CallSC ?
I mean I might know, but Args is not only made for smart contract calls, this seems a bit out of context.